server/routes/scheduler.js

```diff
@@ -13,28 +13,19 @@
  * permissions and limitations under the License.
  */
 
-const response_code = {
-  status: "success",
-  code: 200,
-  message: "Schedule created successfully"
-}
-
-export default function (server) {
-    server.route({
-      path: '/api/reporting/schedule',
-      method: 'POST',
-      async handler(response) {
-        try {
-            await schedule_job()
-          }
-          catch (e) {
-            console.log(e)
-        }
-        return response_code
+export default function(server) {
+  server.route({
+    path: '/api/reporting/schedule',
+    method: 'POST',
+    async handler(response) {
+      try {
+        await schedule_job();
+      } catch (e) {
+        console.log(e);
       }
-    });
+      return "hello client, it's your friend server";
```

Comment:
I wrote this response when I was just testing on my local machine, I think we should try to return a more meaningful API response.

Reply:
will delete